app/controllers/drivers_controller.rb

```diff
@@ -0,0 +1,83 @@
+class DriversController < ApplicationController
+  def index
+    @drivers =Driver.all
+  end
+
+  def show
+    driver_id = params[:id]
+    @driver = Driver.find_by(id: driver_id)
+    if @driver.nil?
+      redirect_to drivers_path
+      return
+    end
+  end
+
+  def new
+    @driver = Driver.new
+  end
+
+  def create
+    @driver = Driver.new(
+        name: params[:driver][:name],
+        vin: params[:driver][:vin],
+        available: true
+    )
+    if @driver.save
+      redirect_to driver_path(@driver)
+      return
+    else
+      render :new
```

Comment:
You should also include a status code. 

```suggestion
      render :new, status: :bad_request
```